crowdsec-docs/unversioned/user_guides/log_centralization.md

```diff
@@ -0,0 +1,248 @@
+---
+id: log_centralization
+title: Log Centralization
+sidebar_position: 10
+---
+
+## Introduction
+
+Our goal is to centralize 2 types of logs:
+ - Nginx logs
+ - SSH auth logs
+
+We'll configure nginx to directly forward the access logs to our central rsyslog server.<br/>
+For the auth logs, we'll configure a local rsyslog on each web server to forward them to our central server.
+
+On the central server, a rsyslog server will receive those logs and write them to files.<br/>
+On this same server, the Security Engine will analyze those logs to detect malicious behaviors in them.<br/>
+Finally, we will have a Firewall Remediation Component running on each of our web server to block the malicious IPs.
+
+
+Our infrastructure will look like this:
+
+
+
+## Rsyslog Server Setup
+
+Let's start by setting up our central rsyslog.
+
+If rsyslog is not installed, you can install it with `apt install rsyslog` (assuming a debian-like distribution).
+
+First step is to configure rsyslog with an UDP listener and a template to write the received logs to disk.
+
+Create the file `/etc/rsyslog.d/10_remote.conf` with the following content:
+```
+# Load the UDP receiver
+module(load="imudp")
+input(type="imudp" port="514")
+
+# Create a template for the logs
+template(name="NginxLogs" type="string" string="/var/log/remote-logs/%HOSTNAME%/nginx.log")
```

Comment:
let's do `/var/log/nginx/%HOSTNAME%-access.log` to avoid the recursive inotify issue ?